tgui/packages/tgui/interfaces/AltitudeControlConsole.jsx

```diff
@@ -45,21 +65,17 @@ export const AltitudeControlConsole = () => {
               disabled={data.alt === 1}
               onClick={() => act('med_alt')}
             >
-              Set to: Medium Altitude
-            </Button>
-          }
-          {
-            <Button
-              fontSize="20px"
-              textAlign="center"
-              fluid
-              disabled={data.alt === 1.5}
-              onClick={() => act('high_alt')}
-            >
-              Set to: High Altitude
+              Set to: Most optimal orbitable altitude
             </Button>
           }
         </Section>
+        <Section title="System Operational Warning">
+          <Box textAlign="center" fontSize="20px">
+            The Ship will always be set according to the disabled option, if
+            none of the two options available to you are disabled, that means
+            the ship is on high altitude
+          </Box>
+        </Section>
```

Comment:
The information contained here is not a warning, nor is relevant. If you insist on keeping an information block here (that doesn't even display on the interface without scrolling), it should contain information relevant such as when overheating occurs and or what happens when overheating. Or the effects that even changing altitude even does.